spec/arns_spec.lua

```diff
@@ -802,7 +862,7 @@ describe("arns", function()
 		end)
 
 		describe("getPriceForAuctionAtTimestamp", function()
-			it("should return the correct price for an auction at a given timestamp for a permabuy", function()
+			it("should return the correct price for an auction at a given timestamp for permanently owned", function()
```

Comment:
The find and replace all makes the grammar on some of these kinda funny.

Reply:
addressed in [da8ab39](https://github.com/ar-io/ar-io-network-process/pull/156/commits/da8ab39db7266c47d5652789cbbb17c4af73464b) and [a2bf822](https://github.com/ar-io/ar-io-network-process/pull/156/commits/a2bf8227d1ca5341b12c161c34eaf7ab35628ab2)